src/main/java/fact/io/hdureader/FITSStream.java

```diff
@@ -43,7 +43,9 @@ public void init() throws Exception {
         //get calibration constants they are stored in the first (and only) row of this hdu.
         HDU offsetHDU = fits.getHDU("ZDrsCellOffsets");
         if (offsetHDU != null){
-            OptionalTypesMap<String, Serializable> row = ZFITSHeapReader.forTable(offsetHDU.getBinTable()).getNextRow();
+            ZFITSHeapReader reader = ZFITSHeapReader.forTable(offsetHDU.getBinTable());
+            boolean ignoreWrongTileHeader;
```

Comment:
Yes. If I was to read this code I would find a random `true` argument to that  function very surprising. I think the variable name helps.